source/extensions/filters/http/cache/cache_headers_utils.cc

```diff
@@ -0,0 +1,160 @@
+#include "extensions/filters/http/cache/cache_headers_utils.h"
+
+#include <array>
+#include <string>
+
+#include "envoy/common/time.h"
+
+#include "absl/algorithm/container.h"
+#include "absl/strings/ascii.h"
+#include "absl/strings/numbers.h"
+#include "absl/strings/str_split.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace HttpFilters {
+namespace Cache {
+
+std::ostream& operator<<(std::ostream& os, const OptionalDuration& duration) {
+  return duration.has_value() ? os << duration.value().count() : os << " ";
+}
+
+// TODO: clean or remove
```

Comment:
Can we resolve this TODO before merge or should we make a ticket to track the work we need to do?

Reply:
Oh I already did, but forgot to remove the TODOs. Will remove them.